CMakeLists.txt

```diff
@@ -213,6 +213,7 @@ if(opencoarrays_aware_compiler)
   add_mpi_test(get_with_offset_1d 2 ${tests_root}/unit/send-get/get_with_offset_1d)
   add_mpi_test(whole_get_array 2 ${tests_root}/unit/send-get/whole_get_array)
   add_mpi_test(strided_get 2 ${tests_root}/unit/send-get/strided_get)
+  add_mpi_test(strided_sendget 4 ${tests_root}/unit/send-get/strided_sendget)
```

Comment:
Oversubscribing in general is not doing harm, but it does not test whether the test runs with the smallest number of images it is expected to run with. So, to do this correctly one would need to set up tests:
- with less than the **n**umber **o**f **i**mages **r**equired by the test (noir),
- exactly the noir,
- with noir + 1 image, and
- with significantly more images than noir.

So, do you really want to do this for every test?